source/form.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <link rel="icon" type="image/png" sizes="32x32" href="img/icon/favicon.png">
+    <link rel="stylesheet" href="css/">
+    <title>HTML Academy: Кэт энерджи</title>
+  </head>
+  <body>
+    <h1 class="visually hidden" lang="en">CAT ENERGY</h1>
+    <header class="header">
+      <a class="header__logo logo">
+        <img class="logo__img" width="33" height="37" alt="Наш маскот." src="img/icon/logo_mobile.svg">
+      </a>
+      <a class="header__logo-main logo-main">
+        <img class="logo-main__img" width="101" height="18" alt="Логотип «Кэт Энерджи»." src="img/icon/logo_main.svg">
+      </a>
+      <button class="header__toggle" type="button" aria-expanded="false" aria-controls="main-nav">
+        Открыть меню
+      </button>
+      <nav class="header__popover menu" id="main-nav">
+        <ul class="menu__content-list">
+          <li class="menu__content-item">
+            <a class="menu__content-link" href="index.html">Главная</a>
+           </li>
+          <li class="menu__content-item">
+            <a class="menu__content-link" href="catalog.html">Каталог продукции</a>
+          </li>
+          <li class="menu__content-item">
+            <a class="menu__content-link">Подбор программы</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="form">
+        <h2 class="form__title">Подбор программы</h2>
+        <p class="form__text">Заполните анкету, и мы подберем программу питания для вашего кота</p>
+        <form class="form__filter filter" action="https://echo.htmlacademy.ru/" method="get">
+          <fieldset class="filter__about">
+            <label class="filter__about-label" for="name">Имя:*</label>
+            <input class="filter__about-field" type="text" name="name" id="name" placeholder="Барсик" required>
+            <label class="filter__about-label" for="weight">Вес (кг):*</label>
+            <input class="filter__about-field" type="text" name="weight" id="weight" placeholder="7" required>
+            <label class="filter__about-label" for="age">Возраст (лет):</label>
+            <input class="filter__about-field" type="number" name="age" id="age" placeholder="7">
+          </fieldset>
+          <fieldset class="filter__radio">
+            <input class="filter__radio-field" type="radio" name="weight-loss" id="weight-loss" checked>
+            <label class="filter__radio-label" for="weight-loss">Похудение</label>
+            <input class="filter__radio-field" type="radio" name="weight-gain" id="weight-gain">
+            <label class="filter__radio-label" for="weight-gain">Набор массы</label>
+            <input class="filter__radio-field" type="radio" name="unknow" id="unknow">
+            <label class="filter__radio-label" for="unknow">Не знаю (Нужен ваш совет)</label>
+          </fieldset>
```

Comment:
Не работают как радио кнопки (должен выбираться только один вариант)
```suggestion
          <fieldset class="filter__radio">
            <legend class="visually-hidden">Программа</legend>
            <input class="filter__radio-field" type="radio" name="program" value="loss" id="weight-loss" checked>
            <label class="filter__radio-label" for="weight-loss">Похудение</label>
            <input class="filter__radio-field" type="radio" name="program" value="gain" id="weight-gain">
            <label class="filter__radio-label" for="weight-gain">Набор массы</label>
            <input class="filter__radio-field" type="radio" name="program" value="advice"  id="unknow">
            <label class="filter__radio-label" for="unknow">Не знаю (Нужен ваш совет)</label>
          </fieldset>```